src/Sulu/Component/Content/Repository/Content.php

```diff
@@ -322,14 +322,14 @@ public function getChildren()
     }
 
     /**
-     * @return Row
+     * @return RowInterface
      */
     public function getRow()
     {
         return $this->row;
     }
 
-    public function setRow(Row $row)
+    public function setRow(RowInterface $row)
```

Comment:
if the bc break is an issue, you could still keep Row here, as the jackalope row does implement RowInterface.

but eventually it would be best if no Jackalope references need to be in sulu code.